src/module.c

```diff
@@ -6461,6 +6461,24 @@ const char *RM_CallReplyProto(RedisModuleCallReply *reply, size_t *len) {
     return callReplyGetProto(reply, len);
 }
 
+int RM_EndExecutionUnit() {
```

Comment:
This does not really ends the execution unit right? It is just flush whatever is pending. Or maybe we can think about it as end the current execution unit and immediately start a new one? So maybe we can call it `RM_ExecutionUnitDrain`?

Reply:
i'm not happy with the name, but maybe while writing the doc comment a new name will reveal itself.
also, we're missing the flags argument i suggested so that in the future we'll be able to let the module control what it does (if we'll need to).
and also probably missing a context argument for future use.